src/@next/AuthenticationInput/AuthenticationInputSingle.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useRef } from 'react';
+import { NumberInput } from '../NumberInput';
+
+interface AuthenticationInputSingleProps
+  extends Omit<React.HTMLAttributes<HTMLInputElement>, 'onChange'> {
+  index: number;
+  value: number | '';
```

Comment:
i cannot understand this, why do we have an empty string here? cant we just the string type?

Reply:
the value can be empty string or a number, since our OTP is a number